tezos-sig-gating-contracts/scripts/utils/helper.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
```

Comment:
why do we have a .js file here 🤔 
You should probably ignore all js files in gitignore

Reply:
Yes I agree , this file should have been ignored